src/ui/buttons/props.js

```diff
@@ -807,6 +881,10 @@ export function normalizeButtonProps(
     }
   }
 
+  if (message) {
+    validateButtonMessage(props, message);
+  }
```

Comment:
Should we follow the same pattern as the `style` below?
`message = validateButtonMessage(props, message);`